metronome/logging/src/io/iohk/metronome/logging/LogTracer.scala

```diff
@@ -0,0 +1,38 @@
+package io.iohk.metronome.logging
+
+import cats.syntax.contravariant._
+import cats.effect.Sync
+import io.circe.syntax._
+import io.iohk.metronome.tracer.Tracer
+import org.slf4j.LoggerFactory
+
+/** Forward traces to SLF4J logs. */
+object LogTracer {
+
+  /** Create a logger for `HybridLogObject` that delegates to SLF4J. */
+  def hybrid[F[_]: Sync]: Tracer[F, HybridLogObject] =
+    new Tracer[F, HybridLogObject] {
+      override def apply(log: => HybridLogObject): F[Unit] = Sync[F].delay {
+        val logger = LoggerFactory.getLogger(log.source)
+
+        def message = s"${log.message} ${log.event.asJson.noSpaces}"
+
+        log.level match {
+          case HybridLogObject.Level.Error =>
+            if (logger.isErrorEnabled) logger.error(message)
```

Comment:
If we would https://github.com/lightbend/scala-logging here, we could probably avoid those `logger.isLevelEnabled` checks

Reply:
Yeah but it does it in the background via macros. I thought using this directly we can avoid the extra machinery that puts in line numbers and source classes or whatever, because they are irrelevant/unknown.